controllers/accesslogpolicy_controller.go

```diff
@@ -133,16 +140,24 @@ func (r *accessLogPolicyReconciler) reconcileUpsert(ctx context.Context, alp *an
 			alp.Spec.TargetRef.Kind, alp.Spec.TargetRef.Name)
 		err := r.updateAccessLogPolicyStatus(ctx, alp, gwv1alpha2.PolicyReasonTargetNotFound)
 		if err != nil {
```

Comment:
Seems we have this behavior: `If the targetRef does not exist, the controller will enter a reconciliation loop until the targetRef exists or the Policy is updated/deleted`

One thing I am not sure is we return `retryErr==nil` with policy status `Accepted==false` with reason `TargetNotFound`, does the controller can keep the reconciliation loop?

Reply:
My bad, I worded this poorly. Correct to

```
If the VPC Lattice resource corresponding to the targetRef does not exist, the controller will enter a reconciliation loop until the targetRef exists or the Policy is updated/deleted
```

If the targetRef itself does not exist, we just set the status to TargetNotFound. If the Service Network or Service does not exist, then we do the retry loop.

Reply:
Great, thanks for clarifying! 

For the behavior: `If the targetRef itself does not exist, the status of the Access Log Policy is set to TargetNotFound`

If the AccessLogPolicy k8s resource is created first, then later, corresponding targetRefObj (Gateway or HTTPRoute or GRPCRoute) is created, do we have a sort of `gwEventHandler.MapToAccessLogPolicy()` (`enqueueImpactedAccessLogPolicy()`) to make the controller do the `(r *accessLogPolicyReconciler) reconcile()` again? Assume you will do it in a separate task _[EKS][Access Logs] Handle Gateway and Route_ ?

Reply:
For the case of reverse order creation, the reconciliation loop should handle it for us without trouble